fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -89,6 +90,8 @@ def self.run(params)
             ['auto-update', value]
           when :notify
             [key, value]
+          when :timeout
+            [key, value]
```

Comment:
Instead of mapping `:timeout` here, can we reject `:timeout` like we do with `:upload_url`? `:timeout` isn't a client option so it shouldn't be mapped in `client_options` 🙃

And then below here call `self.upload_build` like...
```rb
self.upload_build(params[:upload_url], path, client_options, params[:timeout])
```